features/webcryptoapi.yml

```diff
@@ -1,7 +1,12 @@
-draft_date: 2024-08-12
-name: Web Cryptography API
-description: TODO
-spec: https://w3c.github.io/webcrypto/
+name: Web Cryptography
```

Comment:
Also, you'll need to change the file name to `web-crypto`. We try to avoid common suffixes and prefixes, when it still makes sense (e.g., "CSS", "JS", "API" etc.—"Web" is the hardest to avoid, as it's often distinguishing as in "WebUSB").

Reply:
I'd already changed it to webcrypto.yml, which is the "shortname" pretty much everywhere, will that do?

Reply:
per comment below, resolving